sci-mathematics/gismo/gismo-0.8.1.ebuild

```diff
@@ -0,0 +1,37 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=5
+
+inherit cmake-utils eutils
+
+DESCRIPTION='C++ library for geometric design and numerical simulation'
+HOMEPAGE="https://gs.jku.at/gismo"
+SRC_URI="https://github.com/filiatra/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="MPL-2.0"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="doc examples"
+
+# Unbundling in progress, 
+# preparing local changes to get upstream
+
+
```

Comment:
multiple empty lines. Are you using repoman? it should have been complaining about that.